trunk/SUAVE/Components/Energy/Processes/Thrust.py

```diff
@@ -180,7 +185,7 @@ def compute(self,conditions):
         Isp              = Fsp*a0*(1+bypass_ratio)/(f*g)
 
         #Computing the TSFC
-        TSFC             = 3600.*f*g/(Fsp*a0*(1+bypass_ratio))  
+        TSFC             = 3600.*f*g/(Fsp*a0*(1+bypass_ratio))*(1.-SFC_adjustment)
```

Comment:
Should not have 3600 in here, right?   should be secs-to-hour conversion or something...